prow/config.yaml

```diff
@@ -22,8 +22,23 @@ log_level: info
 
 tide:
   queries:
-  - "type:pr state:open repo:kubernetes/test-infra repo:kubernetes/federation label:lgtm label:approved -label:needs-ok-to-test -label:do-not-merge/work-in-progress -label:do-not-merge/hold label:\"cncf-cla: yes\""
-  - "type:pr state:open repo:kubernetes/kubectl review:approved -label:do-not-merge/work-in-progress -label:do-not-merge/hold"
+  - repos:
+    - kubernetes/test-infra
+    - kubernetes/federation
+    labels:
+    - lgtm
+    - approved
+    - "cncf-cla: yes"
+    missingLabels:
+    - needs-ok-to-test
+    - do-not-merge/work-in-progress
```

Comment:
Currently we have:

```
do-not-merge/blocked-paths
do-not-merge/cherry-pick-not-approved
do-not-merge/hold
do-not-merge/release-note-label-needed
do-not-merge/work-in-progress
do-not-merge
```

Are some of these being left out on purpose? (I know it predates this PR)

Reply:
Don't think so. I just left it as-is.

Reply:
ACK